VB-Application/src/main/java/com/gmail/visualbukkit/project/JavadocsManager.java

```diff
@@ -0,0 +1,97 @@
+package com.gmail.visualbukkit.project;
+
+import com.gmail.visualbukkit.VisualBukkitApp;
+import com.gmail.visualbukkit.blocks.definitions.core.*;
+import com.gmail.visualbukkit.blocks.parameters.ClassParameter;
+import com.gmail.visualbukkit.blocks.parameters.FieldParameter;
+import com.gmail.visualbukkit.blocks.parameters.MethodParameter;
+import org.json.JSONObject;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+
+public class JavadocsManager {
+
+    public static void getStatJavadocs(ClassParameter classParameter, MethodParameter methodParameter) {
+        getJavadocs(classParameter.getValue().getName(), methodParameter.getValue().getSignature());
+    }
+
+    public static void getExprJavadocs(ClassParameter classParameter, MethodParameter methodParameter) {
+        getJavadocs(classParameter.getValue().getName(), methodParameter.getValue().getSignature());
+    }
+
+    public static void getExprJavadocs(ClassParameter classParameter, FieldParameter fieldParameter) {
+        getJavadocs(classParameter.getValue().getName(), fieldParameter.getValue().getName());
+    }
+
+    public static void getCompJavaDocs(PluginComponent pluginComponent) {
+        switch (pluginComponent.getBlockType().orElse("Unknown")) {
```

Comment:
the plugin component classes should override openJavadocs() instead of this switch